primitives/state-machine/src/ext.rs

```diff
@@ -152,6 +157,15 @@ where
 	B: 'a + Backend<H>,
 	N: crate::changes_trie::BlockNumber,
 {
+
+	fn set_offchain_storage(&mut self, key: &[u8], value: Option<&[u8]>) {
+		const PREFIX : &'static [u8] = b"block-import-info";
```

Comment:
If we add a new prefix here, we need to make it accessible for OCW too. Please alter the `StorageKind` enum and add it there.
For simplicity I'd prefer to put it to `PERSISTENT` storage though than to introduce new variant (with the same semantics as `PERSISTENT`).